package.json

```diff
@@ -13,6 +13,8 @@
   },
   "dependencies": {
     "@babel/core": "7.21.0",
+    "@babel/preset-env": "7.20.2",
+    "@babel/types": "7.21.2",
```

Comment:
Babel is used at runtime at some point, so we do not rely on all of it's tooling. Despite this being not relevant to this very PR, would you mind to open a follow-up PR describing the necessity of having these dependencies as well?

Reply:
Removed from this PR

Reply:
See PR #12.